static/app/components/profiling/flamegraphSearch.tsx

```diff
@@ -74,15 +89,17 @@ function frameSearch(
   }
 
   for (let i = 0; i < fuseResults.length; i++) {
-    const frame = fuseResults[i];
-
-    results[
-      `${
-        frame.item.frame.name +
-        (frame.item.frame.file ? frame.item.frame.file : '') +
-        String(frame.item.start)
-      }`
-    ] = frame.item;
+    const fuseFrameResult = fuseResults[i];
+    const frame = fuseFrameResult.item;
+    const frameId = getFlamegraphFrameSearchId(frame);
+    results[frameId] = {
+      frame,
+      // matches will be defined when using 'includeMatches' in FuseOptions
+      matchIndices: fuseFrameResult.matches!.reduce((acc, val) => {
+        acc.push(...val.indices);
+        return acc;
+      }, [] as Bounds[]),
```

Comment:
Can we reuse the Fuse.RangeTuple type here? Bounds sounds to me like a very generic name 😅